<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScopeFoundry</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>scopefoundry_logo2_1024.png</normaloff>scopefoundry_logo2_1024.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="col_splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>20</number>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QScrollArea" name="quickaccess_scrollArea">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="quickaccess_scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>398</width>
          <height>736</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </widget>
      </widget>
      <widget class="QGroupBox" name="lq_trees_groupBox">
       <property name="minimumSize">
        <size>
         <width>325</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Logged Quantities</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTreeWidget" name="hardware_treeWidget">
           <column>
            <property name="text">
             <string>Hardware</string>
            </property>
           </column>
          </widget>
          <widget class="QTreeWidget" name="measurements_treeWidget">
           <column>
            <property name="text">
             <string>Measurements</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>App</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="sample_lineEdit"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="save_dir_lineEdit"/>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="save_dir_browse_pushButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="save_dir_label">
             <property name="text">
              <string>Save Dir:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="sample_label">
             <property name="text">
              <string>Sample:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QMdiArea" name="mdiArea">
       <property name="viewMode">
        <enum>QMdiArea::TabbedView</enum>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsMovable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1105</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_set_data_dir"/>
    <addaction name="separator"/>
    <addaction name="action_load_ini"/>
    <addaction name="action_auto_save_ini"/>
    <addaction name="action_save_ini"/>
   </widget>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>Advanced</string>
    </property>
    <addaction name="action_console"/>
    <addaction name="action_log_viewer"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_docs"/>
    <addaction name="separator"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="action_save_window_positions"/>
    <addaction name="action_load_window_positions"/>
    <addaction name="separator"/>
    <addaction name="window_action"/>
    <addaction name="tab_action"/>
    <addaction name="cascade_action"/>
    <addaction name="tile_action"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAdvanced"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_tab_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tab Mode</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="action_set_data_dir">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set Data Save Directory ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_load_ini">
   <property name="text">
    <string>&amp;Load Settings ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save_ini">
   <property name="text">
    <string>&amp;Save Settings As ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_console">
   <property name="text">
    <string>&amp;Console </string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="action_docs">
   <property name="text">
    <string>ScopeFoundry Docs</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
  </action>
  <action name="action_log_viewer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Log Viewer</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="action_subwindow_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Subwindow Mode</string>
   </property>
  </action>
  <action name="window_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sub&amp;window Mode</string>
   </property>
   <property name="shortcut">
    <string>Alt+W</string>
   </property>
  </action>
  <action name="tab_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Tab Mode</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="cascade_action">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="tile_action">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="action_auto_save_ini">
   <property name="text">
    <string>Save Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_window_positions">
   <property name="text">
    <string>Save Window Positions</string>
   </property>
  </action>
  <action name="action_load_window_positions">
   <property name="text">
    <string>Load Window Positions</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
